src/WebJobs.Script/Timer/AzureStorageScheduleMonitor.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Azure.WebJobs.Script;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Timers
+{
+    /// <summary>
+    /// <see cref="ScheduleMonitor"/> that stores schedule information in blob storage.
+    /// </summary>
```

Comment:
This is another move, this time from the timer extension right? Comment pointing to the original file would help.